2025-07-10T06:58:08.272 INFO ==STARTUP== Serving on :5555. Time stamps are UTC.
2025-07-10T06:58:14.177 INFO 192.168.8.104 -> PUT /api/v1/telescope/0/connect
2025-07-10T06:58:14.187 INFO 192.168.8.104 -> {'ClientTransactionID': '1', 'ClientID': '4203'}
2025-07-10T06:58:14.198 ERROR DriverException: Telescope.Connect failed
Traceback (most recent call last):
  File "/home/oksanen/AlpycaDevice/device/telescope.py", line 137, in on_put
    print("PUT connect in:",conn, "DevConnected:",DevConnected,"clients:",connected_clients)
NameError: name 'conn' is not defined

